values-hub.yaml

```diff
@@ -166,3 +198,58 @@ clusterGroup:
 #          operator: In
 #          values:
 #            - OpenShift
+
```

Comment:
While it is absolutely technically valid to directly override values in the clusterGroup file like this, we have normally not done this in our patterns. One quirk of the way we construct the framework is that every application in the clusterGroup gets this values file. We have an `extraValueFiles` mechanism that might work better for this case. The contents of this block can be the same, but could potentially be moved to `overrides/sgx-vault.yaml` and then it could be included by changing the vault app block:
```yaml
  vault:
      name: vault
      namespace: vault
      project: hub
      path: common/hashicorp-vault
      extraValueFiles:
        - /overrides/vault-sgx.yaml
```

That would ensure that only the vault application sees those values.

Reply:
Sounds good. I was not aware of `extraVAluesFiles`, so as advised, I moved it to `/overrides` directory

Reply:
That's perfectly understandable. Thanks!